libfreeipmi/include/freeipmi/spec/ipmi-lan-configuration-parameters-spec.h

```diff
@@ -50,13 +50,44 @@ extern "C" {
 #define IPMI_LAN_CONFIGURATION_PARAMETER_RMCPPLUS_MESSAGING_CIPHER_SUITE_PRIVILEGE_LEVELS 24
 #define IPMI_LAN_CONFIGURATION_PARAMETER_DESTINATION_ADDRESS_VLAN_TAGS                    25
 #define IPMI_LAN_CONFIGURATION_PARAMETER_BAD_PASSWORD_THRESHOLD                           26
+#define IPMI_LAN_CONFIGURATION_PARAMETER_IPV6_SUPPORT                                     50
+#define IPMI_LAN_CONFIGURATION_PARAMETER_IPV6_ADDRESSING_ENABLE                           51
```

Comment:
I think the first two should stick to the spec's nomenclature of "IPV6_IPV4", so it's clear these parameters are for IPv6 & IPv4 config (in contrast to later configs that are only for IPv6).